internals/models.py

```diff
@@ -786,7 +787,9 @@ def get_all(self, limit=None, order='-updated', filterby=None,
           # This includes being an owner, editor, or the original creator
           # of the feature.
           query = query.filter(
-            ndb.OR(Feature.owner == comparator, Feature.editors == comparator))
+            ndb.OR(Feature.owner == comparator,
```

Comment:
A single `ndb.OR()` can have any number of arguments, so you don't need to nest.

Reply:
Changed!